content/docs/ui/account-and-settings/mail.md

```diff
@@ -57,6 +57,21 @@ Allows you to set a schedule for SendGrid to automatically delete contacts from
 
 - Hard Bounces - The number of days, after which SendGrid will purge all contacts from your hard bounces suppression lists.
 
+## Bypass
+
+Bypass filters allow you to disregard unsubscribe groups and suppressions to ensure that an email is delivered to all recipients, whether or not they are on a suppression or unsubscribe list. It is important to respect unsubscribes, and these filters should be used only when it is absolutely necessary to deliver a message to recipients who have unsubscribed from your emails.
```

Comment:
If you wanted to add, one example of needing to bypass would be password resets and or confirmations that sometime can get accidentally unsubbed or marked as spam.